main.go

```diff
@@ -65,27 +71,27 @@ func main() {
 
 	/* Manages traversal threads, which go through the tree of dependencies
 	 * And spawn new Go routines for each new node in the tree. */
-	finder := Finder{
-		deps:         make(map[string]bool),
-		fetchers:     NewFetcherPool(workersNum, requestTimeout, repos),
-		ignoreScopes: strings.Split(ignoreScopes, ","),
-		recursive:    recursive,
-	}
+	fnr := finder.New(
```

Comment:
Ah, I forgot to highlight that I change the name of this var. Basically it is good practice not to have a variable name that collides / is the same as an imported package.

Reply:
FYI it was me that introduced the package name collision so just something to make you aware of rather than anything you did.